### PR TITLE
chore(csharp): Add Basic Test for AST Class

### PR DESCRIPTION
**Pull request by noanflaherty**
Adds a very basic test for asserting that a Class definition can be written to a string as expected.

## Summary by CodeRabbit

- **Tests**
	- Introduced a new test for validating C# class generation, specifically for a "Car" class within the "Automotive" namespace.

### CODE REVIEW

File: generators/csharp/codegen/src/ast/__test__/Class.test.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        });
+        expect(
+            clazz.toString({
+                namespace: "",
+                allNamespaceSegments: new Set<string>(),
+                allTypeClassReferences: new Map<string, Set<string>>(),
+                rootNamespace: "",
+                customConfig: {}
+            })
+        ).toEqual(`\
```
Comment (dsinghvi) [inline generators/csharp/codegen/src/ast/__test__/Class.test.ts:31]: @noanflaherty want to update this to be `toMatchSnapshot()` ? that way we'll have a dedicated file with the generated code and itll be easier to track diffs.

Comment (noanflaherty) [reply generators/csharp/codegen/src/ast/__test__/Class.test.ts:31]: Yeah good call. I'll look for examples of `toMatchSnapshot` elsewhere.

**Comment by noanflaherty** [comment]
@dsinghvi this has been updated to use `toMatchSnapshot`. This is my first time using it – quite nice!

**Review by dsinghvi — APPROVED** [review]
🚀

**Pull request merged**